### DISCUSSION TITLE
Bundle menu option: Open link in new tab

### DISCUSSION THREAD

**Discussion by johnchaffee** [thegeeklab/hugo-geekdoc#445] [Ideas]
It would be great if it was possible to add links to the sidebar menu that will open in a new tab (e.g. `target="_blank"`)? 

For example, by adding a target property to the yaml file like so:

```
  - name: Releases
    ref: "https://github.com/thegeeklab/hugo-geekdoc/releases"
    external: true
    target: "_blank"
```

**Comment by xoxys** [comment]
Thanks for your request. Personally Im not a fan of target blank links as that should always be a user preference and there are good ways to control open in new tab/window in every browser. Thats why I dont plan to integrate it in the theme, sorry.

**Comment by johnchaffee** [comment]
No worries. It's good to be opinionated. :) Thanks for the response.
